examples/python/urdf_loader/README.md

```diff
@@ -0,0 +1,46 @@
+<!--[metadata]
+title = "URDF Loader"
+source = "https://github.com/rerun-io/rerun-loader-python-example-urdf"
+tags = ["3D", "mesh", "loader"]
+thumbnail = "https://static.rerun.io/urdf_loader/9c04fbb376cd4f7498628a98593035c6da0f17fb/480w.png"
+thumbnail_dimensions = [480, 480]
+-->
+
+<picture>
+  <img src="https://static.rerun.io/urdf_loader/fe6730519ceb0f73040fce8aa7cc89e773bafe5c/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/urdf_loader/fe6730519ceb0f73040fce8aa7cc89e773bafe5c/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/urdf_loader/fe6730519ceb0f73040fce8aa7cc89e773bafe5c/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/urdf_loader/fe6730519ceb0f73040fce8aa7cc89e773bafe5c/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/urdf_loader/fe6730519ceb0f73040fce8aa7cc89e773bafe5c/1200w.png">
+</picture>
+
+
+## Overview
+
+This is an example data-loader plugin that lets you view URDF files. It uses the [external data loader mechanism](https://www.rerun.io/docs/howto/open-any-file#external-dataloaders) to add this capability to the Rerun viewer without modifying the viewer itself.
```

Comment:
I would be nice if `URDF` was a link somewhere, e.g. https://wiki.ros.org/urdf

Reply:
good call